.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "crates/l2/contracts/lib/forge-std"]
```

Comment:
instead of using submodules, let's clone the repo like with do with hive and ethereum-package

Reply:
Fixed in [70d07d0](https://github.com/lambdaclass/ethereum_rust/pull/570/commits/70d07d004c16045ceaeb2bb218430586219d1b86)

Reply:
Now:
- the libs are installed when running `make init`.
- they were added to the `.gitignore`.
- they are cleaned when running `make clean`.